src/main/java/org/fogbowcloud/manager/core/constants/CommonConfigurationConstants.java

```diff
@@ -0,0 +1,26 @@
+package org.fogbowcloud.manager.core.constants;
+
+public class CommonConfigurationConstants {
+
+	// from the old class ManagerController in fogbow-manager
+	public static final String DEFAULT_COMMON_SSH_USER = "fogbow";
```

Comment:
If _SSH_COMMON_USER_KEY_ is a configuration constant, then your default value should be in the file **DefaultConfigurationConstants**.